tmp/.gitignore

```diff
@@ -0,0 +1 @@
+!.gitignore
```

Comment:
it does need a static file yeah, maybe it would be possible to add a boolean parameter in case someone doesnt need it. Or you could just leave it saved without deleting since it'll most likely be used sooner or later again